dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -252,17 +252,17 @@ function shareProject() {
 }
 
 function remixProject() {
-  delete dashboard.currentApp.id;
-  delete dashboard.currentApp.hidden;
-  dashboard.currentApp.name = "Remix: " + (dashboard.currentApp.name || appOptions.level.projectTemplateLevelName || "My Project");
-  $('.project_name').text(dashboard.currentApp.name);
-  dashboard.saveProject();
+  delete dashboard.project.current.id;
+  delete dashboard.project.current.hidden;
+  dashboard.project.current.name = "Remix: " + (dashboard.project.current.name || appOptions.level.projectTemplateLevelName || "My Project");
+  $('.project_name').text(dashboard.project.current.name);
+  dashboard.project.save();
 }
```

Comment:
Much of the logic here seems like it might better belong in projects.js since it's actually modifying project